### DISCUSSION TITLE
Desactivaron mi zona de actions en mi cuenta de github

### DISCUSSION THREAD

**Discussion by andyjvv** [community/community#128289] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

Lo uso para usar un vps de tiempo limitado pero veo que me acaban de desactivar la zona de actions github

**Comment by andyjvv** [comment]
quiero que me vuelvan a activar github actions, por favor gracias.

**Comment by Irissuu** [comment, marked as answer]
Si no puede encontrar el motivo de la desactivación, comuníquese con el soporte de GitHub. Ellos podrán brindarle información específica sobre el caso y ayudarlo, buena suerte
![Captura de tela 2024-06-12 202215](https://github.com/community/community/assets/161527170/0c27a626-d5a5-4be8-b1d0-ca733ead5dc7)

**Comment by grandmas-favorite** [comment]
Hi @andyjvv, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
